.eslintrc.json

```diff
@@ -1,21 +1,29 @@
 {
-  "parser": "babel-eslint",
-  "extends": ["airbnb", "plugin:prettier/recommended"],
+  "extends": ["airbnb", "prettier", "prettier/react"],
   "plugins": ["prettier"],
+  "rules": {
+    "class-methods-use-this": 0,
+    "react/jsx-filename-extension": [
+      1,
+      {
+        "extensions": [".js", ".jsx"]
+      }
+    ],
+    "prettier/prettier": ["error"]
+  },
   "globals": {
     "window": false,
     "document": false,
     "URLSearchParams": false,
     "fetch": false,
-    "it": false
-  },
-  "rules": {
-    "prettier/prettier": [
-      "error",
-      {
-        "singleQuote": true,
```

Comment:
btw why from single quotes to double ones?

Reply:
So that the rules between eslint and prettier are the same. 
Webstorm is complaining a lot and since single-quotes double-quotes are not really important I thought of changing that. Plus there is no real reason adding this rule there in the first place. 
If you find that this is not nice then we can chat about it.